.circleci/config.yml

```diff
@@ -224,6 +224,28 @@ jobs:
           environment:
             RS_GLOBAL_DTORS: 1
 
+  build_fuzz:
+    docker:
+      - image: redisfab/rmbuilder:6.0.9-x64-bionic
+        environment:
+            LANG: en_US.UTF-8
+            LANGUAGE: en_US.UTF-8
+            LC_ALL: en_US.UTF-8
+    steps:
+      - early_returns
+      - checkout
+      - setup-prerequisits
+      - load-cached-deps
+      - attach_workspace:
+          at: /workspace
+      - run:
+          name: Run 30-second fuzz test
+          command: |
+              make fuzz # Run the fuzz tester
```

Comment:
looking at [CI](https://app.circleci.com/pipelines/github/RedisGraph/RedisGraph/3210/workflows/21a2447b-13ee-4b15-9d78-dc766d6d1a00/jobs/11690) it seems like `make fuzz` build RedisGraph, please try to run the fuzzer on the artifact of the [build](https://app.circleci.com/pipelines/github/RedisGraph/RedisGraph/3210/workflows/21a2447b-13ee-4b15-9d78-dc766d6d1a00/jobs/11692) step, consult with either @chayim or @rafie if you need help here.